src/Speckle.Sdk/Credentials/Account.cs

```diff
@@ -9,13 +8,13 @@ namespace Speckle.Sdk.Credentials;
 [ComVisible(true)]
 public class Account : IEquatable<Account>
 {
-  private string _id;
+  private string? _id;
 
   /// <remarks>
   /// The account id is unique to user and server url.
   /// </remarks>
   /// <exception cref="InvalidOperationException">Account object invalid: missing required info</exception>
-  public string id
+  public string? id
```

Comment:
For now, do you think its possible to keep `Account` and `AccountManager` in a `nullable disable` context?

I have a stash I can polish quickly that would bring some cleanup of these classes.